controllers/src/main/java/com/redhat/ipaas/controllers/integration/StatusChangeHandlerProvider.java

```diff
@@ -37,26 +37,30 @@
 
         class StatusUpdate {
             private Optional<Integration.Status> newStatus;
-            private String message;
+            private Optional<String> message;
```

Comment:
wouldn't it makes sense to move this property to `errorMessage` as it is then clearer what it is used for (at least at the moment) ?